apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -224,7 +222,7 @@ const SlotPicker = ({
       {selectedDate && (
         <AvailableTimes
           isLoading={isLoading}
-          slots={slots[yyyymmdd(selectedDate.toDate())]}
+          slots={slots[selectedDate.format("YYYY-MM-DD")]}
```

Comment:
Doing toDate() here returned client time locale and didn't match with what our server returned